CUE4Parse/UE4/Assets/Exports/StaticMesh/UStaticMesh.cs

```diff
@@ -63,8 +63,16 @@ public override void Deserialize(FAssetArchive Ar, long validPos)
                 var bHasSpeedTreeWind = Ar.ReadBoolean();
                 if (bHasSpeedTreeWind)
                 {
-                    Ar.Position = validPos;
-                    return;
+                    //Skipping FSpeedTreeWind
+                    if (Ar.Ver >= EUnrealEngineObjectUE4Version.SPEEDTREE_WIND_V7) 
+                    {
+                        Ar.Position += 1460;
+                    } else
+                    {
+                        Ar.Position += 1448;
+                    }
+                    //Ar.Position = validPos;
+                    //return;
```

Comment:
sounds better to just [parse SpeedTreeWind](https://github.com/EpicGames/UnrealEngine/blob/46544fa5e0aa9e6740c19b44b0628b72e7bbd5ce/Engine/Source/Runtime/Engine/Private/SpeedTreeWind.cpp#L748) like [so](https://github.com/EpicGames/UnrealEngine/blob/46544fa5e0aa9e6740c19b44b0628b72e7bbd5ce/Engine/Source/Runtime/Engine/Private/StaticMesh.cpp#L5192)

you don't have to do it, I'm just pasting links so that it won't get lost